### DISCUSSION TITLE
weston中如何使用fcitx5的问题

### DISCUSSION THREAD

**Discussion by maxdianwei** [fcitx/fcitx5#742] [Q&A] [answered]
我想在weston的代码中，调用fcitx5的接口输入unicode到应用中，例如wps。
我最少需要安装哪些库呢

**Comment by wengxt** [comment, marked as answer]
我不太理解你的问题，这个问题和 weston 就没什么关系。首先 wps 使用 Qt，但没有带 qt wayland，所以只能用 Xwayland，用 Xwayland 就和 weston 本身关系不大了。

其次，因为是闭源软件，WPS 的 im module 是 bundle 在 wps 包当中的。所以只需要设置 QT_IM_MODULE=fcitx 来启动 wps 即可。

**Reply by maxdianwei** [reply]
我搭建的环境是参考微软的wslg，weston(backend-rdp) + freerdp + Xwayland。freerdp应用端，会把远端输入法输入的中文，转换成对应的unicode编码，通过freerdp协议传递到weston，然后我在weston目前通过调用xdotool来输入中文，目前发现会有丢字或者错字的问题。
所以想尝试，是否fcitx有接口可以传入unicode编码，输出中文到应用窗口。
我不知道我这样描述是否有清晰的表达。
